apps/theme/pageBuilder/styles/elements/image.scss

```diff
@@ -11,9 +11,11 @@ img {
 .webiny-pb-page-element-image {
   overflow: hidden;
```

Comment:
I think this has to be removed as well. Try inserting an image 100x100px in the editor, and assign a shadow with a spread of 5px to the image. You'll see the shadow is only visible on the sides, and gets hidden on the top and bottom. 
<img width="162" alt="image" src="https://user-images.githubusercontent.com/3808420/201639469-98cad485-add6-44ea-b71f-30087ae58e8d.png">